content/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -13,179 +13,188 @@ description: >-
+1.  Alert all members of the organization about its impending deletion.
+1. Cancel in progress and pending runs or wait for them to finish. Terraform Cloud cannot change the name of an organization with runs in progress.
+1. Lock all workspaces to ensure that no new runs will start before you change the name.
+1. Rename the organization.
+1. Update all components using the Terraform Cloud API to the new organization name. This includes Terraform's `cloud` block CLI integration, the `tfe` Terraform provider, and any external API integrations.
+1. Unlock workspaces and resume normal operations.
 
-To rename a new organization with few or no collaborators, make sure there are no runs in progress and then update the name on the general settings page. Renaming a brand new organization is generally harmless and non-disruptive.
+#### Plan & Billing
 
-To rename an organization that is already managing significant infrastructure, you must:
+Review the organization's plan and any invoices for previous plan payments. Organization owners can also upgrade to one of Terraform Cloud's paid plans, downgrade to a free plan, or begin a free trial of paid features.
 
-1. Alert all members of the organization ahead of time, so they're prepared for the disruption.
-1. Lock all workspaces, and bring all current and pending runs to an end (by cancelling them or by waiting for them to finish). For your protection, Terraform Cloud won't change the name of an organization with runs in progress; locking workspaces ensures no new runs will start before you attempt to change the name.
-1. Rename the organization.
-1. Update anything that uses Terraform Cloud's API with the organization to use its new name. This includes Terraform's `remote` backend, the `tfe` Terraform provider, and any external API integrations.
-1. Unlock workspaces and resume normal operations.
+#### Tags
+
+Review a list of tags for all resources across the organization. When you delete a tag from this page, Terraform Cloud removes it from all resources.
 
-### Plan & Billing
+#### Teams
 
-The plan and billing page allows organization owners to upgrade to one of Terraform Cloud's paid plans, downgrade to a free plan, or begin a free trial of paid features. It also displays any invoices for previous plan payments.
+-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade plan. Free organizations only include an owners team that can include up to five members. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 
-### Tags
+All users in an organization can access the **Teams** page, which displays a list of [teams][] within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
```

Comment:
```suggestion
All users in an organization can access the **Teams** page, which displays a list of teams within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
```

I wasn't sure if those brackets should be included or not.

Reply:
ahh that's a fair question! This is a weird linking pattern that is kind of legacy throughout the docs. So in this case yes, it should be. They define what this link is at the top of the page and then we can reuse it throughout without having to re-enter the link path each time (it's pretty convenient actually).